raglweb/__init__.py

```diff
@@ -109,17 +111,26 @@ def scoreboards():
             status = ''  # TODO: handle late status
 
         rows.append(dict(
-            row_id=len(rows) + 1,
+            # row_id=1
             profile_id=profile_id,
             name=profile_name,
             avatar_url=avatar_url,
             played=nb_played,
             max_matches=max_matches[division],
-            wins=wins,
-            losses=losses,
-            winrate=wins / nb_played * 100 if nb_played else 0,
+            wins=wins + forfeit_wins,
+            losses=losses + forfeit_losses,
+            winrate=(wins + forfeit_wins) / nb_played * 100 if nb_played else 0,
             status=status or '',
         ))
+
+        # Take into account forfeit games when sorting the list
+        rows = sorted(rows, key = lambda row:(row["wins"], row["winrate"]), reverse=True)
+        # Set player rank based on sorted list
+        i = 1
+        for r in rows:
+            r['row_id'] = 'SF' if "Season Forfeit" in r['status'] else i
+            i += 1
```

Comment:
I don't understand: you aare setting the same key several times?

Btw, you can avoid the `i` counter by doing `for i, r in enumerate(row, 1): ...`

Reply:
The original ranking does not take into account the wins/losses from forfeit games. I re-order the list after adding in those wins and then explicitly add the "row_id" attribute which is only used to display the ranking in the scoreboard table. Players that forfeit the entire season should not be actually ranked but just display "SF".

Does that clarify it?

Reply:
Yes, thanks